### PR TITLE
Dcat issue 1526 bis annette proposal

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
This implements suggestions made by  @agreiner on Pr #1578.
@agreiner: A huge thank for revising Pr #1578. I like that now the section provides more explicit guidance than before, so I am inclined to accept your proposal.  

@davebrowning, @pchampin, @andrea-perego (Who had accepted PR#1578 already): are you comfortable with this rephrasing? Please provide your GIT Review.

### CODE REVIEW

File: dcat/index.html [hunk 1]
```diff
@@ -5977,14 +5977,15 @@ <h2>DCAT Profiles</h2>
...
-      and as such defines terms that may be related to personal information.  In addition, it also supports the association of <a href="#Property:resource_rights">rights</a> and <a href="#Property:resource_license">licenses</a> with cataloged Resources and Distributions.
-      These rights and licenses could potentially include or reference sensitive information such as user and asset identifiers as <a data-cite="?ODRL-VOCAB#privacy-consideration">described</a> in [[!ODRL-VOCAB]].   Implementations that produce, maintain, publish or
-      consume such vocabulary terms must take steps to ensure security and privacy considerations are addressed at the application level.
+    The DCAT vocabulary supports the datasets that may contain personal or private information. In addition, the metadata expressed with DCAT may itself contain personal or private information, such as resource <a href="#Property:resource_creator">creators</a>, <a href="#Property:resource_publisher">publishers</a> and other parties or agents via <a href="#qualified-forms">qualified relations</a>.
```
Comment (pchampin) [inline dcat/index.html]: It seems to be that "the" could/should be removed here (but I'm no native speaker, so I may be wrong).
```suggestion
    The DCAT vocabulary supports datasets that may contain personal or private information. In addition, the metadata expressed with DCAT may itself contain personal or private information, such as resource <a href="#Property:resource_creator">creators</a>, <a href="#Property:resource_publisher">publishers</a> and other parties or agents via <a href="#qualified-forms">qualified relations</a>.
```

**Review by pchampin — APPROVED** [review]

File: dcat/index.html (hunk 1 above)
Comment (andrea-perego) [reply dcat/index.html]: +1

Comment (agreiner) [reply dcat/index.html]: Do you mean to accept only the changes to the first paragraph? There's a lot more that differs in my suggestion. (sorry)

**Comment by riccardoAlbertoni** [comment]
@agreiner wrote:
> Do you mean to accept only the changes to the first paragraph? There's a lot more that differs in my suggestion. (sorry)

@agreiner -  Sorry...It seems I have forgot to provide the proper preview and diff.  This PR aims at considering all your changes, see the links below. Though pairwise text comparison is not among my best skill  and a double check is more than welcome.

Preview: https://raw.githack.com/w3c/dxwg/dcat-issue-1526-bis-annetteproposal/dcat/index.html#security_and_privacy

Diff: https://services.w3.org/htmldiff?doc1=https://w3c.github.io/dxwg/dcat/&doc2=https://raw.githack.com/w3c/dxwg/dcat-issue-1526-bis-annetteproposal/dcat/index.html#security_and_privacy

**Comment by agreiner** [comment]
I think you did an excellent job!

**Review by agreiner — APPROVED** [review]
I think this is usable now.

**Review by davebrowning — APPROVED** [review]

**Review by dr-shorthair — APPROVED** [review]

**Review by andrea-perego — APPROVED** [review]

File: dcat/index.html [hunk 2]
```diff
@@ -5977,14 +5977,15 @@ <h2>DCAT Profiles</h2>
...
+    <p>Some datasets require assurances of integrity and authenticity (for example, data about software vulnerabilities). For these, checksums can serve as a type of verification. 
+    DCAT borrows the  <a href="#Class:Checksum"><code>spdx:Checksum</code></a> class from [[!SPDX]] to ensure the integrity and authenticity of DCAT distributions. Publishers may provide a checksum value (a hash) and the algorithm used to generate the hash for each resource in the distribution. A checksum must, however, be provided via a route that is separate from the data it sums. It may be included in metadata that is provided with the data (e.g., a tarfile that includes a file for the distribution and a file for the metadata that includes a checksum for the distribution file), but if so the checksum, or a checksum for the metadata, must also be provided separately to foil an attacker who would manipulate the checksum along with the data. A checksum provided in DCAT metadata will not provide the expected assurances if the integrity and authenticity of the metadata are not also guaranteed.
+    </p>
```
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
   </p>
```

**Comment by riccardoAlbertoni** [comment]
This Pr implements and relaborates suggestions made in issue #1526

**Pull request merged**